enzyme/Enzyme/Enzyme.cpp

```diff
@@ -357,12 +434,48 @@ class Enzyme : public ModulePass {
     auto FT = cast<Function>(fn)->getFunctionType();
     assert(fn);
 
-    bool sret = CI->paramHasAttr(0, Attribute::StructRet) ||
-                cast<Function>(fn)->hasParamAttribute(0, Attribute::StructRet);
-
     IRBuilder<> Builder(CI);
-
+    std::optional<unsigned int> width;
```

Comment:
Probably better to use the llvm::Optional

Reply:
Or perhaps even better have it be a regular integer, default set to 0 as uninitialized.

Reply:
done